src/index.tsx

```diff
@@ -1,3 +1,5 @@
+require('./fabric/NativeScreensModule'); // It needs to stay here so the TurboModule is registered
```

Comment:
This won't be needed anymore, we can settle with this: https://github.com/software-mansion/react-native-screens/commit/699c13edddc0a20737f244fc8c35a93c68e3057e

Reply:
Just merge in the main

Reply:
Changed in: https://github.com/software-mansion/react-native-screens/pull/2437/commits/4df1fc2408102e323ef6a8c9dc4c731e7d4a785e